go-controller/pkg/ovn/base_network_controller_secondary.go

```diff
@@ -792,3 +793,44 @@ func (oc *BaseSecondaryNetworkController) getNetworkID() (int, error) {
 	}
 	return *oc.networkID, nil
 }
+
+// buildUDNEgressSNAT is used to build the conditional SNAT required on L3 and L2 UDNs to
+// steer traffic correctly via mp0 when leaving OVN to the host
+func (bsnc *BaseSecondaryNetworkController) buildUDNEgressSNAT(localPodSubnets []*net.IPNet, outputPort string,
```

Comment:
this isn't needed for localnet. 

Let's at least ensure this is on the tech debt list to be addressed later.